gh #2: Generate_L1_tests

joyal-06 opened the pull request.

joyal-06 committed:
gh #2: Generate_L1_tests

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/middleware-player-interface?pullRequest=3) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/middleware-player-interface?pullRequest=3) before we can accept your contribution.<br/>**3** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: balasaraswathy-n<br/>:white_check_mark: markbuser<br/>:white_check_mark: rtaruni<br/>:x: joyal<br/>:x: joyal-06<br/><hr/>**joyal** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/middleware-player-interface?pullRequest=3) it.</sub>

joyal-06 committed:
gh #2: Generate_L1_tests

joyal-06 committed:
gh #2: Generate_L1_tests

joyal-06 committed:
gh #2: Generate_L1_tests

markbuser committed:
RDKEMW-5974 Build Player Integration Interface on MacOS

Reason for change: Correct install-middleware.sh errors on MacOS
Test Procedure: Run install-middleware.sh on Ubuntu and MacOS, should build sucessfully.

Risks: Low
Priority: P2

balasaraswathy-n committed:
Merge pull request #8 from rdkcentral/RDKEMW-5974

RDKEMW-5974 Build Player Integration Interface on MacOS

rtaruni committed:
Push unit tests for 3headers

Signed-off-by: Radhakrishnan, Tarunikaa <Tarunikaa_Radhakrishnan@comcast.com>

rtaruni committed:
Merge branch 'rdkcentral:develop' into feature/gh2_Generate_L1_Tests

Commit:
gh rdkcentral#2: Generate_L1_tests

Commit:
gh #2: Generate_L1_tests

Commit:
gh #2: Generate_L1_tests
